### PR TITLE
Release 0.49.2

### PR DESCRIPTION
**Pull request by keiko713**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+
+* Bugfix: Ensure all relation information will be sent out even with a lock
+  - This fixes a bug where we were not sending out relation information of
+    relations encountered locks. Processing a snapshot missing such information
+    was failing
+* Allow pg_stat_statements_reset() to fail with a soft error
+  - This was a hard error previously, which failed the snapshot and the snapshot
+    state did not get persisted, indirectly led to a memory leak
+* Add integrity checks before uploading snapshots
+  - Validate some structural assumptions that cannot be enforced by protobuf
```
Comment (msakrejda) [inline CHANGELOG.md:13]: Maybe we should focus on the user-facing functionality:
```suggestion
  - Ensure that malformed snapshots are caught earlier
```
or something like that.

**Review by msakrejda — APPROVED** [review]

**Comment by seanlinsley** [comment]
#406 is likely going to be included in this release. Example description:

> Bugfix: increase activity snapshot timeout to prevent data loss when monitoring many servers

**Comment by lfittl** [comment]
Additionally, it would be good to merge https://github.com/pganalyze/collector/pull/397 before tagging the release (but can happen after this PR is merged) - that way we can get packages built automatically.

**Comment by seanlinsley** [comment]
looks like @keiko713 dropped my changes 🤔 I guess I'll re-apply

**Comment by keiko713** [comment]
> looks like my changes were dropped 🤔 I guess I'll re-apply

I'm so sorry!! 🤦 forgot to pull before rebasing

**Pull request merged**